envs/monkey_zoo/blackbox/test_blackbox.py

```diff
@@ -77,6 +102,23 @@ def island_client(island):
     yield island_client_object
 
 
+def test_logout(simple_island_client):
+    simple_island_client.login()
+    simple_island_client.logout()
+
+    with pytest.raises(Exception):
+        simple_island_client.get_agents()
+
+    with pytest.raises(Exception):
+        simple_island_client.get_machines()
+
+    with pytest.raises(Exception):
+        simple_island_client.get_island_log()
+
+    with pytest.raises(Exception):
+        simple_island_client.run_monkey_local()
```

Comment:
My concern is that this test is too reliant on `MonkeyIslandClient's` implementation. For example, imagine the implementation is something like
```
class Client
    def __init__(self):
         self._token = None

    def login(self):
        self._token = # do_login()

    def logout(self):
        #do logout()
        self._token = None

    def get_agents(self) -> List[Agent]:
        if self._token is None:
            raise Exception("Can't get agents if you're logged out")
        
        return #do get_agents()
```

In this case, logout can be completely broken on the Island but the test will pass. We need to get the token and manually construct queries with `requests` in this test in order to verify that logout is actually working.